.github/scripts/transform_dbt_test_results.py

```diff
@@ -207,14 +213,14 @@ class TestCategory:
     results for output to a workbook and saving them to a cache."""
 
     # Names of fields that are used for debugging
-    _debugging_fieldnames = [TEST_NAME_FIELD, DOCS_URL_FIELD]
+    possible_debugging_fieldnames = [TEST_NAME_FIELD, DOCS_URL_FIELD]
```

Comment:
Now that we're referencing these attributes outside the context of the class (see below), they shouldn't be marked as private anymore.